CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## 2.35.0-rc0 / 2022-04-06
+
+* [CHANGE] Files *.tmp WAL are deleted when Prometheus is starting. #10317
+* [FEATURE] ServiceDiscovery have now a plugin system. #10495
+* [ENHANCEMENT] Add new flag `--lint`  for the commands `check rules` and `check config`. #10435
+* [ENHANCEMENT] Prometheus is built with Go 1.18. #10501
+* [ENHANCEMENT] Log gomaxprocs messages. #10506
+* [ENHANCEMENT] TSDB: more efficient sorting of postings read from WAL at startup. #10500
+* [ENHANCEMENT] Azure SD: Add metric to track Azure SD failure #10476
```

Comment:
```suggestion
* [ENHANCEMENT] Azure SD: Add metric to track Azure SD failures. #10476
```

Reply:
As far as I can see, the PR also contains a small bugfix to the error handling around getting network interfaces?